build/Dockerfile-sbt

```diff
@@ -1,4 +1,4 @@
-FROM hseeberger/scala-sbt:8u312_1.5.5_2.13.6
+FROM hseeberger/scala-sbt:11.0.14.1_1.6.2_2.13.8
```

Comment:
Thanks for this change @wb14123 

I suggest we use the official sbt Docker images as this one isn't maintained anymore.
I think we can use the same one as the used in zio-protoquill, see https://github.com/zio/zio-protoquill/blob/master/build/Dockerfile-sbt#L1